be/src/formats/csv/json_converter.cpp

```diff
@@ -0,0 +1,44 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "formats/csv/json_converter.h"
+
+#include "column/json_column.h"
+#include "common/logging.h"
+#include "gutil/casts.h"
+#include "types/date_value.hpp"
+
+namespace starrocks::vectorized::csv {
+
+Status JsonConverter::write_string(OutputStream* os, const Column& column, size_t row_num,
+                                   const Options& options) const {
+    auto data_column = down_cast<const JsonColumn*>(&column);
+    std::vector<JsonValue>& pool = const_cast<std::vector<JsonValue>&>(data_column->get_pool());
+    JsonValue b = pool.at(row_num);
+    return os->write(b.get_slice());
```

Comment:
```suggestion
    return os->write(pool[row_num].get_slice());
```
if you use JsonValue, you do a value copy. is this necessary?